internal/core/data/v2/application/reading.go

```diff
@@ -49,3 +49,15 @@ func ReadingsByDeviceName(offset int, limit int, name string, dic *di.Container)
 	}
 	return readings, nil
 }
+
+// ReadingCountByDeviceName return the count of all of readings associated with given device and error if any
+func ReadingCountByDeviceName(deviceName string, dic *di.Container) (uint32, errors.EdgeX) {
+	dbClient := v2DataContainer.DBClientFrom(dic.Get)
```

Comment:
Check DeviceName not empty. Same as all other query APIs

Reply:
Added.